utils/hare-bootstrap

```diff
@@ -117,7 +137,7 @@ abort_if_RC_leader_election_is_impossible() {
         if [[ $node == $(node-name) ]]; then
             ssh=
         else
-            ssh="ssh $node"
+            ssh="ssh_safe"
```

Comment:
don't you need to pass `$node` to `ssh_safe` too?

Reply:
Ah yes. missed this. blunder. here it must be ssh $node or sshsafe with node passed to it. right.

Reply:
Made changes to ssh function.